admin-manual/maintenance/troubleshooting.rst

```diff
@@ -1519,62 +1501,61 @@ Which should return something like the following:
    | 1     | XX | en      |
    +-------+----+---------+
 
-If the value shows ``0`` then the setting is not enabled. However, if it shows 
-``1`` (meaning it is enabled) and you need to disable it, use the following - 
-once again, use the numerical value returned from the first query in place of 
-``XX`` in the example below: 
+If the value shows ``0`` then the setting is not enabled. However, if it shows
+``1`` (meaning it is enabled) and you need to disable it, use the following -
+once again, use the numerical value returned from the first query in place of
+``XX`` in the example below:
 
 .. code-block:: sql
 
    UPDATE setting_i18n SET value=0 WHERE id=XX;
 
 The other reason you migh be getting booted back to the home page whenever you
 try to login is that it's possible login for this installation is disabled via
-a setting in one of AtoM's :ref:`configuration files <customization-config-files>` 
-- specifically, the ``config/app.yml`` file. 
+a setting in one of AtoM's :ref:`configuration files <customization-config-files>`
+- specifically, the ``config/app.yml`` file.
 
-Before you update this, double-check: is your installation using a 2-site 
+Before you update this, double-check: is your installation using a 2-site
 deployment model (e.g. one internal read/write site, and a second public facing
-read-only catalog)? If yes, are you sure you are trying to log into the internal 
+read-only catalog)? If yes, are you sure you are trying to log into the internal
 site, and not the public read-only site?
 
 If you've checked that you're trying to log into the correct site and you want
-to ensure that the ``read_only`` setting is not engaged in your edit site, then 
-you will want to open the ``config/app.yml`` file via the command-line and 
+to ensure that the ``read_only`` setting is not engaged in your edit site, then
+you will want to open the ``config/app.yml`` file via the command-line and
 ensure that the ``read_only`` setting is set to ``false``. See:
 
-* :ref:`config-app-yml` 
+* :ref:`config-app-yml`
 
 .. IMPORTANT::
 
-   There is also an environment variable maintained in ``/etc/php5/fpm/pool.d/atom.conf`` 
-   that can override the one in ``config/app.yml``. We recommend checking both 
+   There is also an environment variable maintained in ``/etc/php5/fpm/pool.d/atom.conf``
```

Comment:
We should also update this path - see my general comment for the correct path in 7.2